Source/Diagnostics/FieldIO.H

```diff
@@ -29,6 +29,12 @@ AverageAndPackScalarField( amrex::MultiFab& mf_avg,
                          const amrex::DistributionMapping& dm,
                          const int dcomp, const amrex::IntVect ngrow );
 
+std::vector<std::uint64_t>
+getVec( const amrex::IntVect& v, bool reverse = false);
+
+std::vector<double>
+getVec( const amrex::Real* v, bool reverse = false );
```

Comment:
Please add short Doxygen string for these two new functions :)

Reply:
done!